Commit Sprint_4 project

BoykoBoyko opened the pull request:
В тестах закомментированы поля: 
// driver = new ChromeDriver(); или
    driver = new FirefoxDriver();

Соответственно можно выбрать в каком браузере запускать.

BoykoBoyko committed:
Commit Sprint_4 project

mkouznetsov-git commented on src/test/java/makeorder/MakeOrderPositiveFlow.java:
```diff
@@ -0,0 +1,110 @@
+    @Parameterized.Parameters
+    public static Object [][] getCredentials(){
+        return new Object[][] {
+                {"Елена", "Бойко", "Варшавка 16", "+79169077854", "Я старалась" },
```
Чем принципиально отличаются эти наборы данных? Какие разные варианты тестируются ими?

mkouznetsov-git commented on src/test/java/makeorder/MakeOrderPositiveFlow.java:
```diff
@@ -0,0 +1,110 @@
+    }
+
+    @Test
+    public void enterWithOrderButtonTop() {
```
Два одинаковых метода, отличаются только кнопкой (верхняя/нижняя) -- параметризуй кнопку и оставь один метод.

BoykoBoyko committed:
Commit №2
Параметризовала кнопку Заказать (2 варианта: наверзху и в середине)
Немного изменила тестовые данные

BoykoBoyko committed:
Commit №3
Почему начал застревать click на дату в календаре так и не поняла, поэтому немного переделала метод и запихнула дату в параметризацию, чтоб наверняка.
Проверила в Хроме и Лисе - нигде не застревает больше.
Main убрала, чтоб глаза не мозолил
